wgengine/magicsock/magicsock_test.go

```diff
@@ -396,7 +396,13 @@ func TestPickDERPFallback(t *testing.T) {
 	tstest.ResourceCheck(t)
 
 	c := newConn()
-	c.derpMap = derpmap.Prod()
+	httpc := &http.Client{}
+	dm, err := tailscale.ProdDERPMap(context.Background(), httpc)
```

Comment:
Should I update this to use something local? Having tests rely on the internet is also annoying in a way.

Reply:
This test doesn't really depend on the internet. It uses derpmap.Prod just as a non-zero set of things to test the random distribution over.

You can synthesize a derpmap here in the test. Or write some function which returns a derpmap with `n` regions.

Reply:
ah so it doesn't matter that it's an actual DERPMap, I'll probably just create a random one then